src/Raven.Server/Documents/Indexes/Persistence/Lucene/Documents/LuceneDocumentConverterBase.cs

```diff
@@ -315,6 +317,14 @@ int HandleArray(IEnumerable itemsToIndex)
                 return newFields;
             }
 
+            if (valueType == ValueType.DynamicDictionary)
+            {
+                instance.Add(GetOrCreateField(path + IsDictionaryFieldSuffix, TrueString, null, null, storage, Field.Index.NOT_ANALYZED_NO_NORMS, Field.TermVector.NO));
```

Comment:
I wonder if we want / have to introduce `_IsDictionary` suffix? What is the benefit?

I see we're using to set `FieldType.IsDictionary` but I don't see _any_ read usage of it

Reply:
followed the same logic as array.